saleor/graphql/api.py

```diff
@@ -37,6 +40,11 @@ class Query(graphene.ObjectType):
     category = graphene.Field(
         Category, id=graphene.Argument(graphene.ID),
         description='Lookup a category by ID.')
+    collection = graphene.Field(
+        Collection, id=graphene.Argument(graphene.ID), slug=graphene.String(),
```

Comment:
Is it possible to query collection by slug?

Reply:
I decided not to query by slug.